docs/content/docs/dev/table/sql/insert.md

```diff
@@ -208,7 +208,9 @@ column_list:
 **COLUMN LIST**
 
 Given a table T(a INT, b INT, c INT), Flink supports INSERT INTO T(c, b) SELECT x, y FROM S. The expectation is
-that 'x' is written to column 'c' and 'y' is written to column 'b' and 'a' is set to NULL (assuming column 'a' is nullable). 
+that 'x' is written to column 'c' and 'y' is written to column 'b' and 'a' is set to NULL (assuming column 'a' is nullable).
+For the connector developers who want to avoid overwriting non-target columns with null values when processing partial column updates,
+this user specified target column list can be found from the {{< gh_link file="flink-table/flink-table-common/src/main/java/org/apache/flink/table/connector/sink/DynamicTableSink.java" name="DynamicTableSink$Context.getTargetColumns()" >}}.
```

Comment:
nit
```suggestion
this user-specified target column list can be found from the {{< gh_link file="flink-table/flink-table-common/src/main/java/org/apache/flink/table/connector/sink/DynamicTableSink.java" name="DynamicTableSink$Context.getTargetColumns()" >}}.
```

Reply:
if the above new version is ok, then this dash will not be added